src/main/java/net/minecraftforge/items/SlotItemHandler.java

```diff
@@ -47,6 +47,13 @@ public void set(@NotNull ItemStack stack)
         ((IItemHandlerModifiable) this.getItemHandler()).setStackInSlot(index, stack);
         this.setChanged();
     }
+    
+    @Override
+    public void initialize(ItemStack stack)
+    {
+        ((IItemHandlerModifiable) this.getItemHandler()).setStackInSlot(index, stack);
+        this.setChanged();
```

Comment:
Sounds good. I was hesitant about it, since I was not too sure about the difference with "set", so there might have been a reason for it being different. But since the default vanilla method is the same as "set", it would be fair to do the same here.

Reply:
Yeah that is what I was unsure about whether or not it makes sense. So then in that case add a comment like there is on set regarding IItemHandlerModifiable

Reply:
Like this?